sandbox/src/routes/availableActionsForUserList.js

```diff
@@ -2,7 +2,7 @@ const mockResponseProvider = require('../services/mockResponseProvider')
 const businessFunctionValidator = require('../services/businessFunctionValidator')
 
 function retrieveAvailableActionsForUserList(request, h) {
-  const allowedBusinessFunctions = ["SERVICE_PROVIDER_CLINICIAN"]
+  const allowedBusinessFunctions = ["SERVICE_PROVIDER_CLINICIAN", "REFERRING_CLINICIAN", "REFERRING_CLINICIAN_ADMIN"]
```

Comment:
In the Story ERSSUP-56903, It mentioned that this endpoint should support SPC, SPCA and RC and RCA. But I can't see the allowed business function SPCA. Is that known issue on the pervious implementation?

Reply:
Currently you cannot use SPCA via APIM - Nigel is working on that story now, so SPCA would actually be confusing here. I know its used in a few places - but I think those are incorrect until e go live with SPCA